src/lib/cpp/tests/test_fossdbmanagerclass.cc

```diff
@@ -170,6 +230,14 @@ CPPUNIT_TEST_SUITE(FoLibCPPTest);
     CPPUNIT_ASSERT_EQUAL(expected, results);
   }
 
+  /**
+   * Test to check bad query
+   * \test
+   * -# Spawn a new DbManager.
+   * -# Call fo::DbManager::queryPrintf() with a corrupted statement.
+   * -# Check if result is a failure.
+   * \todo implement fo::DbManager::setLogFile() and check that errors pass through
+   */
```

Comment:
let's create an issue for this and the next occurrence... (@GMishx ok that the TODO was already there)